app/src/main/java/io/keen/client/android/example/MyActivity.java

```diff
@@ -14,7 +14,9 @@
 import io.keen.client.java.KeenLogging;
 import io.keen.client.java.KeenProject;
 
-public class MyActivity extends ActionBarActivity {
+import android.app.Activity;
```

Comment:
Oh, looks like you should be able to remote line 3 (the ActionBarActivity import) now.